.prettierignore

```diff
@@ -0,0 +1,6 @@
+/dist
+/idea
+/node_modules
+/@types
+/testingP
+*-lock
```

Comment:
seem like we don't have these at the top level

Reply:
I've removed testingP and *-lock.
I do see @types when developing locally, so I guess it should also be ignored